src/components/Tab/Tab.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Tab.css';
+import { TabPropTypes } from '../../proptypes/proptypes';
+
+const Tab = ({
+  tab, onTabSelected, index, activeTab,
+}) => {
+  const className = i => (
```

Comment:
`getClassName` is a better option since it's a function and you have to name it properly